remotehttp.go

```diff
@@ -50,17 +55,37 @@ func _isLocalIP(IP net.IP) error {
 		"fc00::/7",      //RFC 4193: Unique-Local
 		"fe80::/10",     //RFC 4291: Section 2.5.6 Link-Scoped Unicast
 		"ff00::/8",      //RFC 4291: Section 2.7
-	} {
-		_, block, err := net.ParseCIDR(cidr)
-		if err != nil {
-			return err
+	}
+
+	// If we've not already parsed our CIDR range
+	if len(ranges) == 0 {
```

Comment:
Maybe have a sync.Once etc round this?

Reply:
I guess it wouldn't hurt; thanks for the feedback.

Reply:
Mm in worse case if someone uses the transport concurrently from two or more goroutines they might end up creating the map many times or concurrent assign to it which might lead weird behaviour or runtime panic. Another solution could be to do this in init() or just have static global map.